cmake/Toolchain-IOS.cmake

```diff
@@ -0,0 +1,66 @@
+# Basic cmake toolchain file for IOS
```

Comment:
I tried that + unix makefiles as the backend but I'm afraid I could not get it to generate arm code, only x86. On my machine at least it's not as simple as just duplicating the OSX toolchain and changing the architecture. It could be my setup is broken for gcc. Feel free to decline this PR as it doesn't really fit your requirements.